asciimatics/widgets/textbox.py

```diff
@@ -283,6 +301,35 @@ def _reflowed_text(self):
 
         return self._reflowed_text_cache
 
+    @property
+    def hide_cursor(self):
+        """
+        Set to True to stop the cursor from showing. If the cursor is hidden,
+        scrolling will move the entire TextBox instead of just the cursor.
+
+        Defaults to False.
+        """
+        return self._hide_cursor
+
+    @hide_cursor.setter
```

Comment:
Note related to this line, but I can't see how to make a comment at a line thT wasn't changed...  You also need to update the logic for `frame_update_count` so that it also returns 0 when the cursor is hidden.

Reply:
And...  Can you add a line to CHANGES.rst to cover the new properties?